src/main/webapp/jsp/module/purap/IWant.jsp

```diff
@@ -83,6 +83,10 @@
 
         <purap:iWantNotes attachmentTypesValuesFinderClass="${documentEntry.attachmentTypesValuesFinderClass}"
                           defaultOpen="true"/>
+        
+        <c:if test="${not isRegularStep}">

```

Comment:
Lines 52 and 126 use "!" instead of "not" for negation. Should this line also use that construct?